.github/actions/on_host_test/action.yaml

```diff
@@ -0,0 +1,72 @@
+name: On Host Test
+description: Runs on-host tests.
+inputs:
+  os:
+    description: "Host OS (either linux or windows)."
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Cloud SDK
+      uses: google-github-actions/setup-gcloud@v0.6.0
+    - name: Configure Environment
+      id: configure-environment
+      shell: bash
+      run: |
+        set -x
+        if [ "${{inputs.os}}" == 'linux' ]
+        then
+          echo "ARCHIVE_EXTENSION=tar.xz" >> $GITHUB_ENV
+        elif [ "${{inputs.os}}" == 'windows' ]
```

Comment:
We don't have windows on this branch, do we need this?

Reply:
removed.